internal/constants/constants.go

```diff
@@ -276,5 +276,8 @@ const SvcConfigPid = "svc-pid"
 // TrayAppName is the name we give our systray application
 const TrayAppName = "ActiveState Platform"
 
+// LaunchFileMacOS is the .plist file that holds the auto launch information on MacOS
+const LaunchFileMacOS = "com.activestate.platform.state-tray.plist"
```

Comment:
Just add this const to the package it's used in, since it's only used once and the value in question isn't something common that requires centralization.